docs/resources/iotda_batchtask.md

```diff
@@ -0,0 +1,154 @@
+---
+subcategory: "IoT Device Access (IoTDA)"
+---
+
+# huaweicloud_iotda_batchtask
+
+Manages an IoTDA batch task within HuaweiCloud.
+
+-> When accessing an IoTDA **standard** or **enterprise** edition instance, you need to specify the IoTDA service
+  endpoint in `provider` block.
+  You can login to the IoTDA console, choose the instance **Overview** and click **Access Details**
+  to view the HTTPS application access address. An example of the access address might be
+  **9bc34xxxxx.st1.iotda-app.ap-southeast-1.myhuaweicloud.com**, then you need to configure the
+  `provider` block as follows:
+
+  ```hcl
+  provider "huaweicloud" {
+    endpoints = {
+      iotda = "https://9bc34xxxxx.st1.iotda-app.ap-southeast-1.myhuaweicloud.com"
+    }
+  }
+  ```
+
+## Example Usage
+
+```hcl
+variable "task_name" {}
+variable "task_type" {}
+variable "targets_file" {}
+
+resource "huaweicloud_iotda_batchtask" "test" { 
+  name         = var.task_name
+  type         = var.task_type
+  targets_file = var.targets_file
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the IoTDA batch task resource.
+  If omitted, the provider-level region will be used. Changing this parameter will create a new resource.
+
+* `name` - (Required, String, ForceNew) Specifies the batch task name. The length does not exceed `128`, and only
+  combinations of Chinese characters, letters, numbers, and underscores (_) are allowed.
+  Changing this parameter will create a new resource.
+
+* `type` - (Required, String, ForceNew) Specifies the batch task type.  
+  The valid values are as follows:
+  + **createDevices**: Batch create devices task.
+  + **updateDevices**: Batch update devices task.
+  + **deleteDevices**: Batch deletion of devices task.
+  + **freezeDevices**: Batch freeze devices task.
+  + **unfreezeDevices**: Batch unfreeze devices task.
+
+* `space_id` - (Optional, String, ForceNew) Specifies the resource space ID to which the batch task belongs.
+  Changing this parameter will create a new resource.
+
+* `targets` - (Optional, List, ForceNew) Specifies the target for executing batch task, fill in the device ID list here,
+  and support a maximum of `30000` device IDs. This parameter is supported when the `type` is **deleteDevices**,
+  **freezeDevices**, and **unfreezeDevices**. Changing this parameter will create a new resource.
+
+* `targets_filter` - (Optional, Map, ForceNew) Specifies the batch task target filtering parameters.
+  (K, V) format, currently supported K includes group_ids (V fills in group ID array), e.g. **{group_ids = "xxxxxx"}**.
```

Comment:
The description of this line of documentation is ambiguous. 
The documentation states `(V fills in group ID array` but the example given is a string.

Reply:
fixed.